packages/lpa-questionnaire-web-app/Dockerfile

```diff
@@ -1,23 +1,23 @@
-FROM node:14-alpine
-ARG APP_NAME
-ARG BUILD_ID
-ARG VERSION
-WORKDIR /opt/app
-ENV APP_NAME="${APP_NAME}"
-ENV BUILD_ID="${BUILD_ID}"
-ENV VERSION="${VERSION}"
-ENV PORT=3000
-# Do not rely on NODE_ENV - exists for performance reasons only
-ENV NODE_ENV=production
-# This must be built locally from the common package
-COPY --from=common:latest /opt/app ../common
-ADD dist dist
-ADD node_modules node_modules
-ADD package.json package.json
-ADD package-lock.json package-lock.json
-RUN npm prune --production \
-  && npm rebuild \
-  && npm version ${VERSION} --no-git-tag-version --allow-same-version || true
-EXPOSE 3000
-USER node
-CMD [ "npm", "start" ]
+FROM node:14-alpine
```

Comment:
Seems the wrong line-endings were used in the original file, which is why this one shows every line in the diff